GamblersMod/GamblingMachine/GamblingMachine.cs

```diff
@@ -39,6 +39,9 @@ internal class GamblingMachine : NetworkBehaviour
         public float currentGamblingOutcomeMultiplier = 1;
         public string currentGamblingOutcome = GamblingOutcome.DEFAULT;
 
+        // TODO (Think about this better)
+        private Coroutine CountdownCooldownCoroutineBeingRan;
```

Comment:
Needed to cancel this coroutine after the machine cooldown starts again just incase. Could've been part of the issue